src/components/UserField/UserField.js

```diff
@@ -1,24 +1,46 @@
 import React from 'react';
-import './userField.css'
+import './userField.css';
 
 export default class UserField extends React.Component {
     constructor() {
         super();
     }
-    
     render() {
         const { users } = this.props;
         return (
             <ul>
                 {users.map((item, index) => {
                     return (
-                        <li key={index}>
-                            {item.name} - {item.surname} - {item.phone} - {item.email} 
-                            {/* <img src = {this.props.url} alt='something'/> */}
+                        <li className="list" key={index}>
+                            <div>
+                                <img src={item.photo} alt="image" style={{height: "50px"}}/>
```

Comment:
if you need to pass more than one prop to the tag or component you need to separate each prop with newline.
and if your tag have no children you have to write space before  '/>'